include/dlaf/eigensolver/bt_band_to_tridiag/impl.h

```diff
@@ -55,9 +55,9 @@ namespace dlaf::eigensolver::internal {
 namespace bt_tridiag {
 
 template <class T>
-matrix::Tile<const T, Device::CPU> setupVWellFormed(const SizeType b,
-                                                    const matrix::Tile<const T, Device::CPU>& tile_hh,
-                                                    matrix::Tile<T, Device::CPU> tile_v) {
+matrix::Tile<T, Device::CPU> setupVWellFormed(const SizeType b,
```

Comment:
Two reasons, neither of which are a must and I'd be open to changing it back.
1. The function takes a `Tile<T, D>`, i.e. non-const element type. The same tile is then returned as `Tile<const T, D>`. I see this as a bit similar to taking a `T*` and then returning it as `T const*` or even worse returning a `const T` (the latter has no effect anyway). In other words, I think it's artificially restricting what the callee is allowed to do with the returned tile. Since there's only one copy of the tile there's no harm in leaving the element type non-const in the return type.
2. It complicates handling of `ReadOnlyTileSender` and `ReadWriteTileSender`. If this returns a `Tile<const T, D>` it would either have to be turned into a `ReadOnlyTileSender` with unnecessary wrapping, or we'd have to deal with a third sender type `unique_any_sender<Tile<const T, D>>`. There's nothing that prevents us doing this technically. I just don't see it as necessary (at the moment at least).

Does that make sense?